Docs/Configuration.tex

```diff
@@ -5100,6 +5100,13 @@ \subsection{Memory Properties}\label{platforminfomemory}
   When \texttt{CustomMemory} is \texttt{false}, this value is automatically set
   based on Mac product name.
 
+  When \texttt{Automatic} is \texttt{true}, the original value from the firmware
+  will be set if valid (i.e. \texttt{non-zero}). Otherwise, the value from \texttt{OcMacInfoLib}
+  will be set. Under non-Automatic mode (i.e. when \texttt{Automatic} is \texttt{false}),
+  a user-specified value will be set if available. Otherwise, the original value from 
+  the firmware will be set if valid (i.e. \texttt{non-zero}).
```

Comment:
Whether or not the value is valid, it essentially is used, because invalid firmware value equals 0. Please drop the `if valid` part.